cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -450,7 +458,7 @@ func createCVR(
 func getPoolMapFromCVRList(cvrList *apis.CStorVolumeReplicaList) map[string]bool {
 	var poolMap = make(map[string]bool)
 	for _, cvr := range cvrList.Items {
-		poolName := cvr.GetLabels()[string(cstorpoolInstanceLabel)]
```

Comment:
we need to remove this string cstorpoolInstanceLabel

Reply:
Removed